src/librustdoc/clean/inline.rs

```diff
@@ -290,10 +290,12 @@ pub fn build_ty(cx: &DocContext<'_>, did: DefId) -> Option<clean::Type> {
     }
 }
 
+// Passed `DefId` has to be a Trait item/path/reexport.
```

Comment:
I'm not sure whether it could also be a primitive, because of lang items. Maybe @GuillaumeGomez would know?
```suggestion
// Passed `DefId` has to be an ADT (struct/union/enum)
```

Reply:
This looks good to merge once you address this comment.

I just realized this can't be a primitive because it assumes you have a `DefId`. So no need to worry about that bit.

Reply:
where's the call site passing `lang_items` of primitive to `build_impls` ?